app/models/institution_builder.rb

```diff
@@ -800,11 +807,15 @@ def self.build_institution_category_ratings_for_category(version_id, category)
         SUM(CASE #{category} WHEN 4 THEN 1 ELSE 0 END),
         SUM(CASE #{category} WHEN 5 THEN 1 ELSE 0 END),
         SUM(CASE WHEN #{category} IS NULL THEN 1 ELSE 0 END),
+        CASE
+          WHEN COUNT(#{category}) = 0 THEN NULL
```

Comment:
I'm rusty when it comes to case statements but should this be a double equal sign?

WHEN COUNT(#{category}) == 0 THEN NULL?